tools/devtools/src/devtools_app/devtools_window/profiler_tab/profiler_tab.xml

```diff
@@ -10,7 +10,7 @@
           <option t-att-selected="store.eventsTreeView" value="Tree">Tree view</option>
           <option t-att-selected="!store.eventsTreeView" value="List">Events log</option>
         </select>
-        <i title="Collapse All" class="fa fa-list me-1" t-on-click="() => this.store.collapseAll()"></i>
+        <i title="Collapse All" type="button" class="fa fa-list me-2" t-on-click="() => this.store.collapseAll()" t-attf-style="{{store.eventsTreeView ? '' : 'visibility: hidden;'}}"></i>
```

Comment:
why didn't you just do a `t-if`? i guess it is intended to keep the empty space to prevent stuff from moving when switching mode?

Reply:
indeed